examples-gallery/plot_ball_rolling_on_spinning_disc.py

```diff
@@ -0,0 +1,620 @@
+ # %%
+"""
+Ball Rolling on Spinning Disc
+=============================
+
+A uniform, solid ball with radius r and mass :math:`m_b` is rolling on a
+horizontal spinning disc without slipping.
+The disc starts at rest and speeds up like this
+:math:`u_{3}(t) = \\Omega \\cdot (1 - e^{(-\\alpha \\cdot t)})`
```

Comment:
```suggestion
:math:`u_{3}(t) = \\Omega \\cdot (1 - e^{(-\\alpha \\cdot t)})`
```

There should only be single back slashes in LaTeX.